src/ert/gui/main_window.py

```diff
@@ -109,11 +117,23 @@ def __init__(
         self.button_group = QButtonGroup(self.side_frame)
         self._external_plot_windows: list[PlotWindow] = []
 
+        self.setStyleSheet("QWidget { color: black; }")
         if self.is_dark_mode():
             self.side_frame.setStyleSheet("background-color: rgb(64, 64, 64);")
         else:
             self.side_frame.setStyleSheet("background-color: lightgray;")
 
+        if self.is_high_contrast_mode():
+            msg_box = QMessageBox()
+            msg_box.setText(
+                "High contrast mode detected. This is not supported by Ert and features may not work as expected."
```

Comment:
""High contrast mode detected. This is not supported by Ert and some features may not work as expected."